src/templates/_common/components/forms.jsx

```diff
@@ -103,7 +103,7 @@ export const SubmitButton = ({
 }) => {
     const content = <React.Fragment>
         {!no_error &&
-            <p id={msg_id || 'msg_form'} className='error-msg no-margin invisible'></p>
+            <p id={msg_id || 'msg_form'} className={`${is_centered ? 'center-text' : 'gr-row'} error-msg no-margin invisible`}></p>
```

Comment:
added this for that missing `center-text` earlier

Reply:
👍